### DISCUSSION TITLE
Llama-3.2-90B-Vision-Instruct

### DISCUSSION THREAD

**Discussion by jhofseth** [ggml-org/llama.cpp#9729] [General]
Is there going to be llama.cpp support for [Llama-3.2-90B-Vision-Instruct](https://huggingface.co/meta-llama/Llama-3.2-90B-Vision-Instruct)?  Thanks.

**Comment by BetaDoggo** [comment]
Anything multimodal right now is tied up in https://github.com/ggerganov/llama.cpp/pull/9687. Presumably it will get support once that's sorted out.
